### ISSUE TITLE
locales

### ISSUE THREAD

**Issue by nordbit** [uxcore/uxcore#3]
It here a possibility to create locales other than cn/en?

**Comment by eternalsky** [comment]
@nordbit yes, more locales are very welcomed, but we need translator to help us, maybe you can make a pr to the component which you want to support more locales. For example,  there is a [i18n file](https://github.com/uxcore/uxcore-dialog/blob/master/src/i18n.js) in the Dialog, and you can add more locales here.

**Comment by herbertpimentel** [comment]
I want to help, but first we may need english docs, that way any dev can translate from english to they own language.

ps: I loved those components and want so much to use them, but docs (not in english) is a little scary to me from Brazil.
